auditing/meilisearch.go

```diff
@@ -503,3 +507,20 @@ func indexName(prefix string, i Interval) string {
 	indexName := prefix + "-" + time.Now().Format(timeFormat)
 	return indexName
 }
+
+func slicesUnorderedEqual[T comparable](lhs, rhs []T) bool {
```

Comment:
Can you provide a unit test for this?

Reply:
Sure. As it's currently unexported, I either:
* can export it withing auditing (though it does not really belong there)
* need to move it to a different module where it makes sense (`pkg/multisort`?)
* might just write a test in the same `auditing` package to be able to access it

Neither felt right. What do you think? I slightly prefer moving to multisort.

Reply:
Not sure if we should export it. For me it's ok here in this package only and unit tested.